public-html/displaynew.php

```diff
@@ -4,41 +4,42 @@
 use src\book;
 use src\ProjectConfig;
 use src\SessionCookie;
+use template\Menu;
 
 $session = new SessionCookie;
-$session->includeAccess();
 
 $config = new ProjectConfig();
 
 
+
+if (isset($_SESSION["logged_in"])) {
+    $menu = new Menu(basename(__FILE__), $_SESSION["admin"], $_SESSION["uname"]);
+} else {
+    $menu = new Menu(basename(__FILE__));
+}
+
```

Comment:
If this logic is being used everywhere why dont you create a class maybe in SessionCookie and then call em

Reply:
Its not everywhere actually some files are just accessible to the admin and some to just anonymous user. But yes this could be done using TemplateRender

Reply:
mm right, cool